ViAn/GUI/mainwindow.cpp

```diff
@@ -564,17 +581,8 @@ void MainWindow::set_selected_video(MyQTreeWidgetItem *newSelectedVideo) {
 void MainWindow::set_selected(MyQTreeWidgetItem *&selected, MyQTreeWidgetItem *new_selected) {
     if(selected == nullptr) {
         selected = new_selected;
-        QString string = selected->text(0);
-        string.append(QString::fromStdString(ARROW_STRING));
-        selected->setText(0, string);
     } else if (selected != new_selected) {
-        QString string = selected->text(0);
-        string.chop(ARROW_STRING.length());
-        selected->setText(0, string);
         selected = new_selected;
-        string = selected->text(0);
-        string.append(QString::fromStdString(ARROW_STRING));
-        selected->setText(0, string);
     }
```

Comment:
You use the .isSelected that already exists instead of trying to use your own selection.